src/main/java/com/google/devtools/build/lib/runtime/BlazeServerStartupOptions.java

```diff
@@ -481,4 +481,15 @@ public String getTypeDescription() {
               + "Requires Windows developer mode to be enabled and Windows 10 version 1703 or "
               + "greater.")
   public boolean enableWindowsSymlinks;
+
+  @Option(
+      name = "unix_digest_hash_attribute_name",
+      defaultValue = "",
+      documentationCategory = OptionDocumentationCategory.UNDOCUMENTED,
+      effectTags = {OptionEffectTag.CHANGES_INPUTS, OptionEffectTag.LOSES_INCREMENTAL_STATE},
+      help =
+          "The name of an extended attribute that can be placed on files to store a precomputed "
+              + "copy of the file's hash, corresponding with --digest_function. This option "
+              + "can be used to reduce disk I/O and CPU load caused by hash computation.")
```

Comment:
I think this needs to be clearer (and probably longer).

Reply:
@ulfjack any suggestions on how to improve the description?